pkgs/development/libraries/openldap/default.nix

```diff
@@ -1,4 +1,7 @@
-{ stdenv, fetchurl, openssl, cyrus_sasl, db, groff, libtool }:
+{ stdenv, lib, fetchurl, openssl, db, groff, libtool
+, cyrus_sasl
+, withCyrusSasl ? true
```

Comment:
```suggestion
, withCyrusSasl ? true, cyrus_sasl
```

Reply:
you can’t be serious …

Reply:
No, I am just joking and thinking that my review comments are not to be taken serious. sarcasm off.

Reply:
You can also move the input below it. Doesn't matter to much.

Reply:
have it your way…